commit ebdefd9601c017c5a7264a94cee2c7af914bca76
Author: Michael Prokop <mika@grml.org>
Date:   Wed Jun 8 10:19:29 2011 +0200

    Drop installation of localized manpages from upstream Makefile.
    
    Upstream's Makefile currently results in:
    
      E: live-boot: manpage-in-wrong-directory usr/share/man/en/man/live-boot.7..gz
      E: live-boot: manpage-in-wrong-directory usr/share/man/en/man/live-snapshot.1..gz
    
    So lets drop the localized manpages and just install the english ones.

--- a/Makefile
+++ b/Makefile
@@ -66,14 +66,6 @@
 		install -D -m 0644 $${MANPAGE} $(DESTDIR)/usr/share/man/man$${SECTION}/$$(basename $${MANPAGE}); \
 	done
 
-	for LANGUAGE in $(LANGUAGES); \
-	do \
-		for MANPAGE in manpages/$${LANGUAGE}/*; \
-		do \
-			SECTION="$$(basename $${MANPAGE} | sed -e 's|\.|\n|g' | tail -n1)"; \
-			install -D -m 0644 $${MANPAGE} $(DESTDIR)/usr/share/man/$${LANGUAGE}/man$${SECTION}/$$(basename $${MANPAGE} .$${LANGUAGE}.$${SECTION}).$${SECTION}; \
-		done; \
-	done
 
 uninstall:
 	# Uninstalling executables
